### ISSUE TITLE
Is this project alive and kicking?

### ISSUE THREAD

**Issue by graste** [dbellettini/php-eventstore-client#33]
Hi,

having a look at this repository I'm wondering whether it's still active? There's a fork by @tetsuobe with projection management support, that's not been incorporated via PR: https://github.com/tetsuobe/php-eventstore-client – It contains some non-automatically mergable changes in terms of vagrant vs. docker test setup iirc. There's another eventstore client project done by @rayrutjes (php7 only): https://github.com/rayrutjes/php-geteventstore-client

Basically I'm wondering what your take is on those three repositories and where things might by heading for EventStore support in the PHP world. :-)

**Comment by dbellettini** [comment]
Hello @graste, currently I'm using either MongoDB WiredTiger or PostgreSQL as event store so I'm not actively developing this project, but I'm fixing reported bugs and reviewing possibly incoming pull requests

**Comment by rayrutjes** [comment]
Hi @graste, I'd recommend you to use @dbellettini 's repo. Even though my repo is well tested, it hasn't been used in production, so I wouldn't recommend it. I also feel that I would do my implementation differently if I had to do it again. Feels a bit over engineered right now.

**Comment by dbellettini** [comment]
Well neither my library has been used in production 😅

**Comment by graste** [comment]
Thanks for the info! :-)
